src/lib.rs

```diff
@@ -86,7 +86,7 @@ fn unsafe_alloc_buffer(len: usize) -> Vec<u8> {
     ret
 }
 
-pub enum Error {
+pub enum EeiError {
```

Comment:
Any better naming proposal? `EeiE` looks quite weird :)

`EthereumError` ? `EwasmError`?

Reply:
@axic `ApiError`?

Reply:
I think `ApiError` is too generic, if you motivation is not to be generic, then `EwasmError` sounds the best.